include/clap/ext/draft/audio-ports-activation.h

```diff
@@ -39,10 +39,10 @@ typedef struct clap_plugin_audio_ports_activation {
 
    // Activate the given port.
    //
-   // It is only possible to activate on the audio-thread if can_activate_while_processing() returns
-   // true.
+   // It is only possible to activate and de-activate on the audio-thread if
+   // can_activate_while_processing() returns true.
    //
-   // [active ? audio-thread : main-thread]
+   // [can_activate_while_processing ? audio-thread : main-thread]
```

Comment:
This change is wrong.
If the plugin isn't active it has to be called on the main thread.
If the plugin is active either it can't be called or it has to be called on the audio thread.

Reply:
I see, I thought `active` was referring to the port `is_active` flag, but you meant it to refer to the `clap_plugin->activate()` state. For the record, `ext/audio-ports-config.h` uses the following wording:
```c
   // [main-thread,plugin-deactivated]
   bool(CLAP_ABI *select)(const clap_plugin_t *plugin, clap_id config_id);
```